content/en/docs/azure/deploy/install-kubeflow.md

```diff
@@ -87,6 +87,8 @@ Example variables:
 
 ## Kubeflow installation
 
+**Important**: If you wish to deploy Kubeflow on Azure with multi-user authentication and namespace separation you must continue with the instructions for [Authentication using OICD in Azure](/docs/azure/authentication-oidc). The instructions below only apply to a single-user Kubeflow deployment. Such a deployment cannot be upgraded to a multi-user deployment at this time.
```

Comment:
Suggested revision (grammar and clarity): 

**Important**: If you wish to deploy Kubeflow on Azure with multi-user authentication and namespace separation you must continue with the instructions for [Authentication using OICD in Azure](/docs/azure/authentication-oidc). The instructions below only apply to a single-user Kubeflow deployment. Such a deployment cannot be upgraded to a multi-user deployment at this time.

->

**Important**: To deploy Kubeflow on Azure with multi-user authentication and namespace separation, use the instructions for [Authentication using OICD in Azure](/docs/azure/authentication-oidc). The instructions in this guide apply only to a single-user Kubeflow deployment. Such a deployment cannot be upgraded to a multi-user deployment at this time.

Reply:
Done. Thanks.